README.md

```diff
@@ -249,11 +245,6 @@ Currently supported types of artifacts:
 
 See [docs/generic.md](docs/generic.md) for more details.
 
-## Project status
-
-Cachi2 was derived (but is not a direct fork) from [Cachito](https://github.com/containerbuildsystem/cachito) and is
-still in early development phase.
-
```

Comment:
Although I am in favour, various other formulations of the same sentence can be found all around the docs. When I tried to remove them/avoid them I was told we want to reference the origin of this project. So, we either should mention this very briefly at the beginning of this document or get rid of all references in all docs. I'd obviously prefer the latter and forget about the original project since this one is very different, but I'd be okay even with the former, as long as we maintain some kind of consistency.